Dockerfile

```diff
@@ -7,18 +7,22 @@ ENV USER_ID=999 \
 
 ENV LANG=en_US.UTF-8    \
     LANGUAGE=en_US:en   \
-    LC_ALL=en_US.UTF-8  \
-    BUILDPACK_URL=https://github.com/ind9/mesos-buildpack-python#mesos-fix
+    LC_ALL=en_US.UTF-8
 
 RUN apt-get update  \
     && apt-get -y install coreutils curl tar git-core daemontools gcc g++ python
+
 RUN curl --silent http://dl.gliderlabs.com/herokuish/latest/linux_x86_64.tgz \
     | tar -xzC /bin
-RUN /bin/herokuish buildpack install  # Install officially supported buildpacks
+
+RUN /bin/herokuish buildpack install  \
+    && curl -L https://raw.githubusercontent.com/ind9/python-stack/mesos-fix/mesos-fix > /bin/mesos-fix \
```

Comment:
@ashwanthkumar even official images do this. Example - https://github.com/docker-library/postgres/tree/8e867c8ba0fc8fd347e43ae53ddeba8e67242a53/9.5

Shouldn't be a problem ideally. Try `COPY` anyway. It is not recommended to use `ADD`